Geometry/HGCalCommonData/macros/HGCalConvert.C

```diff
@@ -0,0 +1,433 @@
+//////////////////////////////////////////////////////////////////////////////
```

Comment:
Is this code eventually going to be used in central workflows, or will it always be standalone/test code?

Even if the latter, rather than a root macro, I would propose a .cc file located in the test folder (or in the bin folder, compiled to a standalone executable)

Reply:
Done that. Also removed the text files. How to put them in the cms-data area. I have forgotten the procedure. Earlier I put the shower libraries there